lib/tasks/alarm_api.rake

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+namespace :alarm_api do
+  desc "Add a few popular Colorado Cities to get things started"
+  task init_cities: :environment do
+    cities = CSV.read("lib/assets/starting-cities.csv").map do |row| # [city, state]
+      {name: row[0], state: row[1]}
+    end
+    City.create(cities)
+  end
+
+  desc "Get new forecasts for all of our cities"
+  task update_all_forecasts: :environment do
+    error_msg = "Failed to retrieve forcast for city: "
+    City.all.each do |city|
+      logger.fatal error_msg + city.name unless city.retrieve_forecast
```

Comment:
I figured we would want to add logs for API failures.  We can look at these locally and, more importantly, we can retrieve them from heroku using the `heroku logs` command in terminal so long as the person running hte command has the heroku toolbelt command.